sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat_metrics.py

```diff
@@ -97,13 +98,16 @@ def __init__(
         disable_offline_storage: bool,
         long_interval_threshold: int,
         has_credential: bool,
+        distro_version: str = "",
     ) -> None:
         self._ikey = instrumentation_key
         self._feature = _StatsbeatFeature.NONE
         if not disable_offline_storage:
             self._feature |= _StatsbeatFeature.DISK_RETRY
         if has_credential:
             self._feature |= _StatsbeatFeature.AAD
+        if distro_version:
+            self._feature |= _StatsbeatFeature.DISTRO
```

Comment:
So, the exporter has the whole version, but statsbeat will only get a BIT, right?